### DISCUSSION TITLE
import SockJS from 'sockjs-client'; The browser reported an error

### DISCUSSION THREAD

**Discussion by 2512746832** [sockjs/sockjs-client#580] [Q&A]
![QQ截图20220301163802](https://user-images.githubusercontent.com/70892499/156134313-39f9896b-44c2-4865-bece-d62e64d60b79.png)

**Comment by eebrown** [comment]
Do you have the same problem in v. 1.5.2?

**Reply by djolec987** [reply]
I have the exact same problem with version 1.6.1.
